src/style-library/components/pages/ButtonScreen.scss

```
@@ -0,0 +1,39 @@
+	display: inline-flex;
+	align-items: center;
+	justify-content: center;
+	padding: var(--kb-sl-space-10) var(--kb-sl-space-20);
+	border: 1px solid transparent;
+	outline: 1px solid var(--kb-sl-color-gray-200);
+	outline-offset: -1px;
+	font-size: var(--kb-sl-font-size-small);
+	font-weight: var(--kb-sl-font-weight-medium);
+	line-height: var(--kb-sl-line-height-s);
+	white-space: nowrap;
+	// Only the three properties `ButtonScreen.js` sets inline from the preset (background, color,
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)